README.md

```diff
@@ -1,33 +1,33 @@
 # GURU-PR [![Build Status](https://travis-ci.org/guru-pr/gurupr.svg?branch=master)](https://travis-ci.org/guru-pr/gurupr) [![Code Climate](https://codeclimate.com/github/guru-pr/gurupr/badges/gpa.svg)](https://codeclimate.com/github/guru-pr/gurupr) [![Test Coverage](https://codeclimate.com/github/guru-pr/gurupr/badges/coverage.svg)](https://codeclimate.com/github/guru-pr/gurupr) [![Dependency Status](https://img.shields.io/gemnasium/guru-pr/gurupr.svg)](https://gemnasium.com/guru-pr/gurupr)
 
-Site do Grupo de Usuários Ruby do Paraná!
+Site of the Ruby users group from Paraná - Brazil!
 
-Utilizamos:
-- Github para autenticação
+We use:
+- Github for authentication
 - Heroku
 - PostgreSQL
 
-De resto, vamos desenvolvendo aos poucos!
+Otherwise, let's develop in baby steps!
 
-## Instalação normal
+## Common install
 
 ```bash
-cp .env.example .env # altere as chaves caso queira trocar a aplicação
-cp config/database.yml.example config/database.yml # configure seus dados de acesso ao DB
+cp .env.example .env # edit the keys in case you want to change the application
+cp config/database.yml.example config/database.yml # config your database access data
 bundle install
 rake db:setup
 rails server
 ```
 
-## Instalação via docker
+## Docker install
```

Comment:
Following the same idea I've mentioned above, it would be more correct to say `Running on Docker`.